src/test/java/configs/Config.java

```diff
@@ -16,6 +16,7 @@ public class Config {
     private final static String COMPLETED_CHECKOUT = "https://demowebshop.tricentis.com/checkout/completed/";
 
 
+
```

Comment:
если домен поменяется на другой, а эндпоинты останутся, то придётся менять строку в каждой переменной. Лучше переписать значения на 
```suggestion

    private final static String BASE_URL = "https://demowebshop.tricentis.com";
    private final static String BOOKS = BASE_URL + "/books";
    private final static String COMPUTERS = BASE_URL + "/computers";
    ...
```